worlds/sa2b/__init__.py

```diff
@@ -237,17 +261,59 @@ def generate_basic(self):
             musiclist_o = list(range(0, 47))
             musiclist_s = musiclist_o.copy()
             self.multiworld.random.shuffle(musiclist_s)
+            musiclist_o.extend(list(range(47, 78)))
+            musiclist_s.extend(list(range(47, 78)))
+
+            if self.multiworld.sadx_music[self.player].value == 1:
+                musiclist_s = [x+100 for x in musiclist_s]
+            elif self.multiworld.sadx_music[self.player].value == 2:
+                for i in range(len(musiclist_s)):
+                    if self.multiworld.random.choice([True, False]):
```

Comment:
```suggestion
                    if self.multiworld.random.randint(0,1):
```
choice of a list feels a bit overkill for this purpose.